cmd/cloud_sql_proxy/proxy_test.go

```diff
@@ -98,6 +99,12 @@ func TestCreateInstanceConfigs(t *testing.T) {
 			continue
 		}
 		if err != nil {
+			if runtime.GOOS == "windows" {
```

Comment:
Granted this test is already is bad shape with all the boolean values, but would it be better to explicitly mark each test as `supportedOnWindows` such that we don't have to check error strings? That way it would be easy to write:

``` go
if runtime.GOOS == "windows" && !v.supportedOnWindows {
  t.Skip("skipping test for Unix sockets or FUSE")
  continue
}
```

Reply:
That would be much cleaner. Should we do this with the other test too, or is checking the network value fine?

Reply:
We can also use the `!build windows` compile tags to skip tests on Windows

Reply:
That's what's being done for the fuse and limits tests. The problem here is that we don't want to skip all of the tests in the module. Just some of the inputs for the table tests

Reply:
Ah understood - we can also consider just refactoring those to different files. I'll defer to Eno on what makes the most sense organizationally.

Reply:
It would be nice to clean up this test such that we could use build tags. For now, we'll at least have Windows-tests explicitly marked so that when we (read: me) refactor the tests, it's an easy job to move things out as needed.